docs/src/Module.doc.js

```diff
@@ -12,49 +12,201 @@ card(
   <PageHeader
     name="Module"
     description="
-The Module component renders detailed information in the expanded state and summarized information in the collapsed state.
-"
+      A Module is a container that holds content about one subject. Its contents can be visible at all times, or expand and collapse as individual modules or a group of modules.
+    "
   />
 );
 
 card(
   <PropTable
     Component={Module}
+    name="Non-Expandable Module"
```

Comment:
"Non-expandable" is a bit awkward…how would you feel about "static"?

Reply:
lol @ayeshakmaz and I were debating between the two words. I don't know if we should ask Kelly instead to see if she can weigh in on this quickly since this is harder to change.